js/ui/check_box.d.ts

```diff
@@ -72,6 +72,12 @@ export interface dxCheckBoxOptions extends EditorOptions<dxCheckBox> {
      * @public
      */
     value?: boolean | null | undefined;
+    /**
+     * @docid
+     * @default false
+     * @public
+     */
+    allowIndeterminateStateByClick?: boolean;
```

Comment:
better place option in the right place in the alphabet order

Reply:
Oh, thank you for your attention